csrc/scheduler/utils.cpp

```diff
@@ -2436,7 +2436,6 @@ bool revertUseOfInputCacheInResize(
 
 void prepareForMemoryTypePromotion(Fusion* fusion) {
```

Comment:
Could you please add your comment https://github.com/NVIDIA/Fuser/pull/28/files#r1142681108 as a comment in the code? I feel hard to understand the code without that.

Reply:
Added to the header file